backend/src/migrations/20230405072520_add_images.ts

```diff
@@ -0,0 +1,20 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'user_profile';
+
+const ColumnName = {
+    AVATAR_URL: 'avatarUrl',
+};
+
+async function up(knex: Knex): Promise<void> {
+    await knex.schema.dropTableIfExists(TABLE_NAME);
```

Comment:
Why are you going to drop this table?

Reply:
Resolved in [6ccc578](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/314/commits/6ccc578e7713a42dbb64accaa2bbbecc31440000)